packages/ui-tree-browser/src/TreeBrowser/props.ts

```diff
@@ -148,6 +148,7 @@ type CollectionBase = {
 type Collection = CollectionBase & {
   items?: number[]
   collections?: (number | string)[]
+  compareFunc?: (a: any, b: any) => number
```

Comment:
Instead of defining `a` and `b` as `anys`, I think it would beneficial for the users to provide some proper type definitions for the `compareFunc`.
Here is an example what I have in mind:

```ts
// in TreeBrowser/props.ts
// define some helper types:
type CompareCollection = Collection & { type: 'collection' }
type CompareItem = CollectionItem & { type: 'item' }
// this is a sum type, so CompareObject is one of CompareCollection OR CompareItem
type CompareObject = CompareItem | CompareCollection

....
// and then use this sum type for the compareFunc:
compareFunc?: (a: CompareObject, b: CompareObject) => number
...

Doing this the user will have proper ide suggestions and autocompletion.